modules/flowable-cmmn-engine/src/main/java/org/flowable/cmmn/engine/impl/runtime/CaseInstanceHelperImpl.java

```diff
@@ -268,7 +268,7 @@ protected Case getCaseModel(CaseDefinition caseDefinition, CmmnModel cmmnModel)
     protected CaseInstanceEntity initializeCaseInstanceEntity(CommandContext commandContext, CaseDefinition caseDefinition, 
             CmmnModel cmmnModel, Case caseModel, CaseInstanceBuilder caseInstanceBuilder) {
         
-        StartCaseInstanceBeforeContext instanceBeforeContext = new StartCaseInstanceBeforeContext(caseInstanceBuilder.getBusinessKey(), caseInstanceBuilder.getName(),
+        StartCaseInstanceBeforeContext instanceBeforeContext = new StartCaseInstanceBeforeContext(caseInstanceBuilder.getBusinessKey(),caseInstanceBuilder.getBusinessStatus(), caseInstanceBuilder.getName(),
```

Comment:
Thanks for the PR; others may comment on the functionality but my comment is on style:  why sometimes is there a space after the comma and other times not?  For example here there is no space before `caseInstanceBuilder` in `getBusinessKey(),caseInstanceBuilder.getBusinessStatus(),`.   The project standard formatting is to include the space after the comma.   There are multiple occurrences of this type.

Reply:
Hey David, 
thanks for your feedback.